src/main/kotlin/eu/theblob42/idea/whichkey/config/MappingConfig.kt

```diff
@@ -154,15 +154,9 @@ object MappingConfig {
             .map {
                 Pair(it.toList(), keyMapping[it]?.getPresentableString() ?: "no description")
             }
-        // check .ideavimrc if the default VIM actions should be displayed along other mappings (default: false)
-        val showVimActions = when (val value = injector.variableService.getGlobalVariableValue("WhichKey_ShowVimActions")) {
-            null -> false
-            is VimString -> value.asString().toBoolean()
-            else -> false
-        }
         val vimActionsPairs =
             VIM_ACTIONS[mode]?.entries
-                ?.map { it.key to if (showVimActions) it.value else "" }
+                ?.map { it.key to "" }
                 ?: listOf()
```

Comment:
I guess we can completely remove this variable. The default descriptions are part of `whichKeyDescriptions` variable anyway and blank descriptions (which covers this complete map) are filtered out in line 200 of this file